src/app/events/utils/sortEvents.ts

```diff
@@ -27,6 +29,12 @@ export function sortEventsAsc<T extends DateFields>(events: Array<T>) {
   return sortEvents(events, compareAsc)
 }
 
+export function sortNonEmptyEventsAsc<T extends DateFields>(
+  events: NonEmptyArray<T>,
+) {
+  return sortEvents(events, compareAsc) as NonEmptyArray<T>
+}
```

Comment:
Making this suggestion so that both types stay in sync.

The "risk" with assertions is forgetting to update them when making other changes that impact them.

Commit if you like it, it's not blocking.

```suggestion
export function sortNonEmptyEventsAsc<
  T extends DateFields,
  Array extends NonEmptyArray<T>,
>(events: Array) {
  return sortEvents(events, compareAsc) as Array
}
```

Reply:
I love that. Done. Thank you.